sql/moz-fx-data-shared-prod/telemetry_derived/glam_sample_counts_v1/query.sql

```diff
@@ -0,0 +1,184 @@
+WITH all_samples AS (
```

Comment:
So, I know this file you probably templated it off of `glam_user_counts_v1` which works and makes sense. I wanted to mention a nicer approach that was used in the glean setup. There's a [template file](https://github.com/mozilla/bigquery-etl/blob/main/bigquery_etl/glam/templates/view_user_counts_v1.sql) that is used to generate the sql [here](https://github.com/mozilla/bigquery-etl/blob/main/bigquery_etl/glam/generate.py#L247)

I'm not sure how easy it would be to reuse this for desktop because I think it's just used for Glean now. You can try it out but it's not necessary we can still keep this as-is. Just something to keep in mind in the future.